tests/unit/dataactvalidator/test_b11_object_class_program_activity_3.py

```diff
@@ -24,6 +24,12 @@ def test_success(database):
 def test_failure(database):
     """ Test invalid object class code (3 digits) """
 
+    # This should return because if it's '000', '00', '0' a warning should be returned
     op = ObjectClassProgramActivityFactory(object_class='000')
+    assert number_of_errors(_FILE, database, models=[op]) == 1
+
+    op = ObjectClassProgramActivityFactory(object_class='00')
+    assert number_of_errors(_FILE, database, models=[op]) == 1
 
+    op = ObjectClassProgramActivityFactory(object_class='0')
```

Comment:
This will give 3 warnings instead of 1 for each of '0' input. I think it's confusing if there is 3 warnings for one invalid input?